www/src/scss/_overlay.scss

```diff
@@ -61,6 +61,30 @@
     width: auto;
   }
 }
+.url-container {
+  display: inline-block;
+  margin-top: 1vh;
+  padding: 8px;
+
+  & > div {
+    display: inline-block;
+  }
+  .input-wrap {
+    align-items: center;
```

Comment:
This is for `display: flex` you don't need it if your using inline-block everywhere. please remove.